srcpkgs/python3-pywebview/template

```diff
@@ -0,0 +1,23 @@
+# Template file for 'python3-pywebview'
+pkgname=python3-pywebview
+version=3.5
+revision=1
+wrksrc="pywebview-${version}"
+build_style=python3-module
+hostmakedepends="python3-setuptools"
+depends="python3
+$(vopt_if gtk 'python3-cairo python3-gobject webkit2gtk')
+$(vopt_if qt 'python3-PyQt5-webengine qt5-webkit')"
+short_desc="Build GUI for your Python program with JavaScript, HTML, and CSS"
+maintainer="Neel Chotai <neel@chot.ai>"
+license="BSD-3-Clause"
+homepage="https://github.com/r0x0r/pywebview"
+distfiles="https://github.com/r0x0r/pywebview/archive/${version}.tar.gz"
+checksum=d97d5de3217939fcdebe8adaca7cdf4bbd76b9086ebeffdf57e5bfd4549b6de1
+make_check=extended
+
+build_options="gtk qt"
```

Comment:
It would probably better to document the optional deps in e.g. a `README.voidlinux`, `pywebview` also has at least somewhat sane error messages if none of the 3 supported libs are installed